phpBB/phpbb/db/migration/data/v320/remove_profilefield_aol.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v320;
+
+
+class remove_profilefield_aol extends \phpbb\db\migration\migration
+{
+    static public function depends_on()
+    {
+        return array(
+            '\phpbb\db\migration\data\v320\dev',
```

Comment:
This file should be under v32x or even v330 (ask devs), but definitely should depend on some newer migration like `\phpbb\db\migration\data\v32x\v322`.

Reply:
Ill leave it on 320 at the moment until some dev has an idea.

Reply:
Any clue about that single failing test?

Reply:
https://travis-ci.org/phpbb/phpbb/jobs/357035293#L618